zookeeper-contrib/zookeeper-contrib-zooinspector/src/main/java/com/nitido/utils/toaster/Toaster.java

```diff
@@ -44,8 +44,6 @@
 /**
  * Class to show tosters in multiplatform
  *
- * @author daniele piras
```

Comment:
Is it okay to remove this? The license header is kind of ambiguous to me, saying it is copyrighted but with Apache license 2.0?

Reply:
I think this is only a mistake done from the committer who committed this files.
I seems that the rules of the project say that author tags are not allowed.